gulpfile.js

```diff
@@ -535,25 +538,48 @@ function buildExtension(name, version, hasCss, options, opt_extraGlobs) {
     mkdirSync('build');
     mkdirSync('build/css');
     const startTime = Date.now();
-    return jsifyCssAsync(path + '/' + name + '.css').then(function(css) {
-      var jsCss = 'export const CSS = ' + JSON.stringify(css) + ';\n';
-      var jsName = 'build/' + name + '-' + version + '.css.js';
-      var cssName = 'build/css/' + name + '-' + version + '.css';
-      fs.writeFileSync(jsName, jsCss, 'utf-8');
-      fs.writeFileSync(cssName, css, 'utf-8');
-      if (options.compileOnlyCss) {
-        return Promise.resolve();
-      }
-      return buildExtensionJs(path, name, version, options);
-    })
-    .then(() => {
+    return buildExtensionCss(path, name, version, options).then(() => {
       endBuildStep('Recompiled CSS in', name, startTime);
     });
   } else {
     return buildExtensionJs(path, name, version, options);
   }
 }
 
+/**
+ * @param {string} path
+ * @param {string} css
+ * @param {string} version
+ * @param {!Object} options
+ */
+function buildExtensionCss(path, name, version, options) {
+  function writeCssBinaries(name, css) {
+    var jsCss = 'export const CSS = ' + JSON.stringify(css) + ';\n';
+    var jsName = `build/${name}.js`;
+    var cssName = `build/css/${name}`;
+    fs.writeFileSync(jsName, jsCss, 'utf-8');
+    fs.writeFileSync(cssName, css, 'utf-8');
+  }
+  const promises = [];
+  const mainCssBinary = jsifyCssAsync(path + '/' + name + '.css')
+      .then(writeCssBinaries.bind(null, `${name}-${version}.css`))
```

Comment:
Nit: I think this is a bit more readable.
```js
.then(() => writeCssBinaries(`${name}-${version}.css`)) 
```

Reply:
i would need to write it as ``.then(css => writeCssBinaries(`${name}-${version}.css`, css))`` 
to get the equivalent. (I'm ok with rewriting it to this)

Reply:
done.